ADR/ADRXXX-remove-forms-api.md

```diff
@@ -0,0 +1,45 @@
+# ADRxxx: Remove forms-api
+
+Date: 2025-08-19
+
+## Status
+
+> Accepted or Rejected
+>
+> Valid ADR statuses are: "Proposed", "Accepted", "Rejected", "Deprecated", "Superseded by ADRzzz", but since we use the GitHub Pull request mechanism to approve ADRs, "Proposed" isn't applicable.
+
+## Context
+
+In [ADR004] we decided to implement a separate API app so that we could scale the services required for form submissions separately to the services required for building forms.
+
+However, maintaining multiple apps has resulted in increased toil. Having multiple services has also resulted in challenges around designing and maintaining the interface between forms-admin and forms-api.
+
+As the cost of compute is low at the moment, the cost of the additional developer time outweighs any savings due to efficient capacity management.
+
+We are currently refactoring forms-admin and forms-api to work with the v2 API described in [ADR034]. We have an opportunity now as part of this work to make a large architectural change, namely using the forms-admin app to serve the v2 API endpoints needed for forms-runner, and removing the need for forms-admin.
+
+[ADR004]: ./ADR004-store-form-schemas-in-an-api.md
+[ADR034]: ./ADR034-api-v2-for-forms.md
+
+## Decision
```

Comment:
Some thoughts on migrating.

The decision states what the end state will be and I appreciate the step by step outline in this section. Will there be a phased migration (shadow traffic, canary release) or a big-bang switch? Is it worth calling that out specifically in the ADR?

What’s the rollback strategy if forms-admin can’t handle the load or the change introduces regressions? I very much doubt this will be the case but I want to ask the question just in case.